### DISCUSSION TITLE
Seems the PWA link is broken?

### DISCUSSION THREAD

**Discussion by kyttndr** [graphql/graphiql#3676] [Q&A / Support]
Hi team, I tried to install GraphiQL PWA following the link provided by the doc, but it seems the link is broken.. 
> An [installable PWA](https://graphiql-test.netlify.com/webpack/) with plugins ([source](https://github.com/graphql/graphiql/blob/main/examples/graphiql-webpack))

And the link provided is https://graphiql-test.netlify.com/webpack/

Where can I find correct link to install the PWA? 

Thanks!

**Comment by dpnova** [comment]
https://github.com/graphql/graphiql/issues/3754

Is it the link from this ticket?
